templates/about.hbs

```diff
@@ -60,7 +61,7 @@
   </table>
 
   <p>You can still use <a href="https://crates.fyi/">crates.fyi</a> domain name for
-  redirects. <a href="https://crates.fyi/crates/clap"><code>https://crates.fyi/crates/clap</code></a> will
+  redirections. <a href="https://crates.fyi/crates/clap"><code>https://crates.fyi/crates/clap</code></a> will
   redirect to documentation of latest version of clap crate.</p>
```

Comment:
Isn't this just saying that the crates.fyi domain will redirect to docs.rs? If so, then saying that would be clearer.